src/azul/chalice.py

```diff
@@ -267,6 +267,10 @@ def route(self,
                 methods = kwargs['methods']
                 self.non_interactive_routes.update((path, method) for method in methods)
             methods = kwargs.get('methods', ())
+            if method_spec:
+                import azul.openapi.spec
```

Comment:
This also smells.

Please dissolve `CommonEndpointSpecs` into `AzulChaliceApp`. That should happen as the first commit in this PR. The methods from `CommonEndpointSpecs` should be appended at the end of `AzulChaliceApp`.